client/src/navigator/Drawer/DrawerMenu.js

```diff
@@ -47,12 +47,13 @@ const DrawerMenu = ({ navigation }) => (
         My Courses
       </Text>
     </View>
-    <View style={styles.head}>
+    <View>
       <FontIcon.Button
         name="times"
-        size={20}
+        size={25}
         color={colors.gray.dark}
-        backgroundColor="white"
+        backgroundColor={"transparent"}
+        underlayColor={"#00000020"}
```

Comment:
I'd suggest moving the "#00000020" color to `client/theme/colors.js`. You can make it a variant of gray.